kotlin/change-basemaps/README.md

```diff
@@ -1,18 +1,28 @@
-# Change Basemaps
-### Category: Layers
-![Change Basemaps App](change-basemaps.png)
+# Change basemaps
 
-The Change Basemaps sample app shows how you can change basemaps from an Android Navigation Drawer.  
+Change a map's basemap. A basemap is beneath all layers on an `ArcGISMap` and is used to provide visual reference for the operational layers.
 
-## Features
+![Image of change basemap](change-basemaps.png)
+
+## Use case
+
+Basemaps should selected contextually, for example, in maritime applications, it would be more appropriate to use a basemap of the world's oceans as opposed to a basemap of the world's streets.
```

Comment:
same advice as in java version:

small grammar typo (missing the verb "to be"). I broke the sentence into two as well because it was a bit of a run-on.
```suggestion
Basemaps should be selected contextually. For example in maritime applications it would be more appropriate to use a basemap of the world's oceans, as opposed to a basemap of the world's streets.
```